lib/ykpiv.c

```diff
@@ -96,6 +96,12 @@ static unsigned char *set_object(int object_id, unsigned char *buffer) {
   return buffer;
 }
 
+static void scrub_and_free_pin(char *pin) {
+  int len = strlen(pin) * sizeof(char);
```

Comment:
this should probably be a size_t since that's what strlen() returns and memset() expects